src/components/keypad/button.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import {StyleSheet} from "aphrodite";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+
+type Props = {
+    onPress: () => void,
```

Comment:
The suggestion is for callback to be typed as returning `mixed`: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/128975067/Flow+Best+Practices#Callback-props-whose-return-types-aren-t-used-should-be-mixed-when-possible